test/phantom-lib.js

```diff
@@ -7,11 +7,12 @@
 
 localStorage.clear();
 
-if (!window.Sizzle)
-    throw "Sizzle was not properly loaded"
-
 function ph_select(sel) {
-    return Sizzle(sel);
+    var i, els = [];
+    var list = document.querySelectorAll(sel);
+    for (i = 0; i < list.length; i++)
```

Comment:
note that according to https://developer.mozilla.org/en-US/docs/Web/API/NodeList#A_.22live.22_collection
we can also use `Array.prototype.slice.call(div_list);` here
Would that be less compatible than the simple loop?